examples/audio_classifier/raspberry_pi/README.md

```diff
@@ -0,0 +1,58 @@
+# MediaPipe Audio classifier example with Raspberry Pi
+
+This example uses [MediaPipe](https://github.com/google/mediapipe) with Python on
+a Raspberry Pi to perform real-time audio classification using audio streamed
+from the microphone.
+
+## Set up your hardware
+
+Before you begin, you need to
+[set up your Raspberry Pi](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up)
+with Raspberry Pi OS (preferably updated to Buster).
+
+Raspberry Pi doesn't have a microphone integrated on its board, so you need to
+plug in a USB microphone to record audio.
+
+## Install PortAudio and MediaPipe
+
+This project requires the installation of the PortAudio library and MediaPipe for running inference.
+
+You can install these dependencies by using the provided script.
+
+```
+sh setup.sh
+```
+
+## Download the examples repository
+
+First, clone this Git repo onto your Raspberry Pi like this:
+
+```
+https://github.com/googlesamples/mediapipe.git
```

Comment:
We can skip giving the link here since we're not actually showing the 'like this' part. Just mention that they will need to clone the repo.

Reply:
I'll just post this comment once since it should apply to all of the rpi readmes.

Reply:
Fixed it.